Bottom: 8b57543a3a5dc8e27ef9e2cd3366c58231720f4e
Top:    8bc69cc85f5ae9c4288d7ebda349e8b41ca26782
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 04:19:34 +0000

Refresh of find-c-kill-setthing-with-fire

---

diff --git a/src/find.c b/src/find.c
index 327a96a..18385d7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -401,11 +401,11 @@ PinLineIntersect (PinType *PV, LineType *Line)
 bool
 SetThing (int type, void *ptr1, void *ptr2, void *ptr3)
 {
+  assert (! (type == PIN_TYPE && ptr1 == NULL));
   thing_ptr1 = ptr1;
   thing_ptr2 = ptr2;
   thing_ptr3 = ptr3;
   thing_type = type;
-  assert (! (type == PIN_TYPE && ptr1 == NULL))
   return true;
 }
